packages/synapse-react-client/src/components/FeaturedDataTabs/FacetPlotsCard.tsx

```diff
@@ -189,18 +234,22 @@ const FacetPlotsCard: React.FunctionComponent<FacetPlotsCardProps> = ({
                     )}
                   </SizeMe>
                   <div className="FacetPlotsCard__body__legend">
-                    <FacetPlotLegend
+                    <FacetPlotLegendTable
+                      facetName={
+                        getColumnDisplayName(facetDataArray[index].columnName)!
+                      }
```

Comment:
Determines the table column header (in this case, "Data Type")

![image](https://github.com/Sage-Bionetworks/synapse-web-monorepo/assets/17580037/df5a1f4f-e892-469f-a7dd-c2a44a5f2d9d)